website/docs/docs/use-dbt-semantic-layer/sl-faqs.md

```diff
@@ -0,0 +1,256 @@
+</expandable>
+
+<expandable alt_header="Do you support semi-additive metrics?">
+
+Yes, we approach this by specifying a [dimension](/docs/build/dimensions) that a metric cannot be aggregated across (such as `time`). You can learn how to configure semi-additive dimensions [here](/docs/build/measures#non-additive-dimensions).
+</expandable>
+
+<expandable alt_header="Can I use an entity as a dimension?">
+
+Yes, while [entities](/docs/build/entities) must be defined under “entities,”  they can be queried like dimensions in downstream tools. Additionally, if the entity isn't used to perform joins across your semantic models, you may optionally define it as a dimension.
+</expandable>
+
+## Available integrations
+
+<expandable alt_header="What analytics tool integration are supported today?">
+
+There are a number of data applications have integrations with the dbt Semantic Layer, including Tableau, Google Sheets, Hex, and Mode, among others.
+
+Refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more information.
+
+</expandable>
+
+<expandable alt_header="How can I benefit from using the dbt Semantic Layer if my visualization tool is not currently supported?">
+
+You can use [exports](/docs/use-dbt-semantic-layer/exports) to materialize your metrics into a table or view in your data platform. From there, you can connect your visualization tool to your data platform. 
+
+Although this approach doesn't provide the dynamic benefits of the dbt Semantic Layer, you still benefit from centralized metrics and from using MetricFlow configurations to define, generate, and compute SQL for your metrics.
+
+</expandable>
+
+<expandable alt_header="Why should I use exports as opposed to defining a view within my data platform?">
+
+Creating an [export](/docs/use-dbt-semantic-layer/exports) allows you to bring your governed metric definitions into your data platform as a table or view. This means your metric logic is managed centrally in dbt, instead of as a view in your data platform and ensures that metric values remain consistent across all interfaces.
+
+</expandable>
+
+<expandable alt_header="Can metric descriptions be viewed from third-party tools?">
+
+Yes, all of our interfaces or APIs expose metric descriptions, which you can surface in downstream tools.
+
+</expandable>
+
+## Permissions and access
+
+<expandable alt_header="How do fine-grained access controls work with the dbt Semantic Layer?">
+
+Currently, the credentials you configure when setting up the dbt Semantic Layer are used for every request. Any physical access policies you have tied to your credentials will be respected.
+
+We are currently working on introducing more fine-grained access controls, including user-level access and group credentials, that enable flexible granular permissions.
+</expandable>
+
+## Availability
```

Comment:
i left this hear but my gut instinct was to move this above (after overview) in case if users  have immediate questions about availability and compatibility.